### DISCUSSION TITLE
Use on a terminal server

### DISCUSSION THREAD

**Discussion by sofdwere** [NickeManarin/ScreenToGif#1227] [Q&A] [answered]
Can ScreenToGif be easily installed on a terminal server like Windows Server 2019 and then launched and used by different logged-in users at the same time?

**Comment by NickeManarin** [comment, marked as answer]
You should install to "Everyone":

![image](https://github.com/NickeManarin/ScreenToGif/assets/14798947/5643f2eb-2ef3-4cb1-9333-5e1970686722)

Then I don't remember if you need to, but if the app complains about multiple instances, you can change the setting here:

![image](https://github.com/NickeManarin/ScreenToGif/assets/14798947/bfcae0c2-1a5e-4cd1-8b5b-9249214e4650)
